ansible/fastly.yml

```diff
@@ -10,8 +10,7 @@
       fastly_service:
         name: "{{ item.service }}.theforeman.org"
         fastly_api_key: "{{ fastly_api_key }}"
-        domains:
-          - name: "{{ item.service }}.theforeman.org"
+        domains: "{{ item.domains | default([{'name': item.service ~ '.theforeman.org'}]) }}"
```

Comment:
What's `~` here?

Reply:
String concat.

Reply:
https://jinja.palletsprojects.com/en/stable/templates/#other-operators

Reply:
I guess that and tried to look for that but somehow I couldn't find it.

Reply:
documenting things properly is complicated, as you know :)